production/helm/loki/values.yaml

```diff
@@ -221,6 +224,12 @@ loki:
   # -- Check https://grafana.com/docs/loki/latest/configuration/#schema_config for more info on how to configure schemas
   schemaConfig: {}
 
+  # -- Check https://grafana.com/docs/loki/latest/configuration/#ruler for more info on configuring ruler
+  rulerConfig:
+    alertmanager_url: ""
+    enable_alertmanager_v2: false
+    enable_api: true
```

Comment:
```suggestion
  rulerConfig: {}
```

Reply:
Good suggestion, was thinking there may be a better way to approach it myself. 👍